aws/check-alb-target-group-health.yaml

```diff
@@ -0,0 +1,87 @@
+---
+# Description
+#   Check the health of Application Load Balancer (ALB) target groups
+#
+# Instructions
+#   1. Add the "aws" subscription to agents that should run this check.
+#   2. More configuration options are available for this plugin; please see the
+#      plugin documentation for more details.
+#
+# Documentation
+#   - Usage: https://github.com/sensu-plugins/sensu-plugins-aws#usage-examples
+#
+# Contributors
+#   The following individuals have contributed to this configuration template:
+#   - Todd Campbell, @nixwiz
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: check-alb-target-group-health
+spec:
+  command: >-
+    check-alb-target-group-health
+    --aws_region={{ .annotations.check_alb_target_group_health_aws_region | default "us-east-1" }}
+    --target_groups={{ .annotations.check_alb_target_group_health_target_groups | default "target-group-1" }}
+  runtime_assets:
+    - sensu-plugins/sensu-plugins-aws:0.0.5
+  publish: false
+  subscriptions:
+    - aws
```

Comment:
Here's another example of a pattern we ought to experiment with (see comments on PR #33 RE: Zookeeper subscriptions). It's impractical to use a single subscription for `aws`, so nesting these templates in subdirectories by AWS service, and using more descriptive subscription names like `aws/alb` seems like a better pattern. Thoughts?

Reply:
I nested these similar to the 'system/linux' subscriptions.  So there is in an overall 'aws' subscription, but also a per service based subscription (e.g. 'aws/ec2', 'aws/elb').